test/contentset.test.js

```diff
@@ -0,0 +1,270 @@
+
+    await expect(result instanceof Promise).toBeTruthy()
+    await expect(result).rejects.toEqual(
+        new codes.ERROR_DELETE_CONTENTSET({ messageValues: 'https://cloudmanager.adobe.io/api/program/123/contentSet/321 (404 Not Found)' })
+    )
+})
+
+test('deleteContentSet', async () => {
+    expect.assertions(2)
+
+    const sdkClient = await createSdkClient()
+    const result = sdkClient.deleteContentSet('1234', '456')
+
+    await expect(result instanceof Promise).toBeTruthy()
+    await expect(result).resolves.toEqual({})
+})
+
+test('getContentSet - not found', async () => {
+    expect.assertions(2)
+
+    const sdkClient = await createSdkClient()
+    const result = sdkClient.getContentSet('123', '321')
+
+    await expect(result instanceof Promise).toBeTruthy()
+    await expect(result).rejects.toEqual(
+        new codes.ERROR_GET_CONTENTSET({ messageValues: 'https://cloudmanager.adobe.io/api/program/123/contentSet/321 (404 Not Found)'})
+    )
+})
+
+test('getContentSet', async () => {
+    expect.assertions(2)
+
+    const sdkClient = await createSdkClient()
+    const result = sdkClient.getContentSet('123', '1234')
+
+    await expect(result instanceof Promise).toBeTruthy()
+    await expect(result).resolves.toMatchObject({
+        name: 'One path',
+        description: 'A description',
+        paths: [{
+          path: '/content/dam/seed19017.png'
+        }],
+        programId: '123'
+    })
+})
+
+test('updateContentSet - not found', async () => {
+    expect.assertions(2)
+
+    const sdkClient = await createSdkClient()
+    const result = sdkClient.updateContentSet('1234', '6402', {})
+
+    await expect(result instanceof Promise).toBeTruthy()
+    await expect(result).rejects.toEqual(
+        new codes.ERROR_UPDATE_CONTENTSET({ messageValues: 'https://cloudmanager.adobe.io/api/program/1234/contentSet/6402 (404 Not Found)'})
+    )
+})
+
+test('updateContentSet', async () => {
+    expect.assertions(2)
+
+    const sdkClient = await createSdkClient()
+    const result = sdkClient.updateContentSet('67403', '6402',  require('./jest/data/contentset-update.json'))
+
+    // Verify the updated represntation is returned.
+    await expect(result instanceof Promise).toBeTruthy()
+    await expect(result).resolves.toMatchObject(require('./jest/data/contentset-update.json'))
+})
+
+
+test('listContentSets', async () => {
+    expect.assertions(2)
+
+    const sdkClient = await createSdkClient()
+    const result = sdkClient.listContentSets('123456')
+
+    await expect(result instanceof Promise).toBeTruthy()
+    await expect(result).resolves.toMatchObject([
+        {
+            id: '6402',
+            name: 'One path two ptah',
+            description: "A description",
+        },
+        {
+            id: '6352',
+            name: 'One path',
+            description: "Another description",
+        }])
+})
+
+// Content Flow tests
+
+test('createContentFlow not found ', async () => {
```

Comment:
You could create a content-set.test.js and content-flow.test.js. And the tests from each file could be structured under a  main `describe('Content Set ...', () => { tests... })`. And each action can have its own `describe`, like `describe('Create content flow', () => { tests ... })`.

Reply:
Makes sense to separate them out.  Kept them together since the API definitions for both are in same file.